.github/workflows/hierarchy.yml

```diff
@@ -112,7 +112,6 @@ jobs:
     needs:
       - diff
       - dprint-format
-      - license-check
```

Comment:
Why not?

Reply:
Because that job only runs for `isRust = true`